g4f/Provider/__init__.py

```diff
@@ -43,20 +40,16 @@
 from .GptGod          import GptGod
 from .GptTalkRu       import GptTalkRu
 from .Hashnode        import Hashnode
+from .HuggingChat     import HuggingChat
```

Comment:
Consider maintaining alphabetical order for consistency with the rest of the imports.